string id
geometry_msgs/Pose current_pose
---
bool status
